src/main/java/org/radarbase/appserver/exception/EmailMessageTransmitException.java

```diff
@@ -1,5 +1,9 @@
 package org.radarbase.appserver.exception;
 
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
 public class EmailMessageTransmitException extends MessageTransmitException {
```

Comment:
Are the `MessageTransmitException` (s) ever unhandled? You don't need this response status here if they are caught and handled. Moreover, these are thrown when a scheduled job is run, not in response to an HTTP request, so again, it does not make sense to add these here.